aether-kernel/aether/kernel/api/models.py

```diff
@@ -414,6 +415,7 @@ class SchemaDecorator(ExportModelOperationsMixin('kernel_schemadecorator'), Proj
     transport_rule = models.TextField(null=True, blank=True, verbose_name=_('transport rule'))
     masked_fields = models.TextField(null=True, blank=True, verbose_name=_('masked fields'))
     is_encrypted = models.BooleanField(default=False, verbose_name=_('encrypted?'))
+    topic_name = JSONField(null=True, blank=True, verbose_name=_('topic name'))
```

Comment:
Does it need to be a JSON field, could it be a simple TextField?

Reply:
@shawnsarwar ?

Reply:
Let's use a JSONField in case we want to put more advanced production options here.